MIGRATING.md

```diff
@@ -2,6 +2,34 @@
 
 This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
 
+## Migrating from 4.x to 5.x
+
+In previous versions, null resource properties would result in a resource object with all its properties set to `null`. Now, null resource properties would result in `null` property value.
+
+_Before:_
+
+```
+{                                                 deserializedResource.Prop1.Should().Be("Hello World!");          
+    prop1 : "Hello World!",         =>            deserializedResource.NestedObject.Should().NotBeNull();
+    nestedObject: null                            deserializedResource.NestedObject.Prop1.Should().BeNull();
+}
+
+```
+
+_Now:_
+
+```
+{                                                 deserializedResource.Prop1.Should().Be("Hello World!");          
+    prop1 : "Hello World!",         =>            deserializedResource.NestedObject.Should().BeNull();
+    nestedObject: null                            
+}
+
+```
+
+Since this is a breaking change in the default behavior, Okta.Sdk was published with version numbers starting from 5.0.0.
```

Comment:
```suggestion
Since this is a breaking change in the default behavior, the major version of Okta.Sdk was incremented to 5; the latest version is now 5.0.0.
```